carbon-design-system/carbon
# fix(TableToolbarSearch): TableToolbarSearch & Search placeholder prop

Closes https://github.com/carbon-design-system/carbon/issues/7500

Deprecates `placeHolderText` in favor of `placeholder` to align better with our other Form elements

#### Changelog

**New**

- `placeholder` prop to align with other elements like `TextInput`

**Changed**

- Added `deprecate` blocks to the `placeHolderText` PropType definitions in `Search` and `TableToolbarSearch`

#### Testing / Reviewing

- Ensure you can use `placeholder` on `Search` and `TableToolbarSearch`, and no proptype warnings appear
- Ensure you can use `placeHolderText` and proptype warnings _do_ appear
